templates/libraries/jersey3/pojo.mustache

```diff
@@ -82,6 +82,10 @@ public class {{classname}} {{#parent}}extends {{{.}}} {{/parent}}{{#vendorExtens
   private {{{datatypeWithEnum}}} {{name}}{{#required}}{{#defaultValue}} = {{{.}}}{{/defaultValue}}{{/required}}{{^required}} = null{{/required}};
   {{/isContainer}}
   {{^isContainer}}
+  {{#description}}
+
+  // {{{.}}}
```

Comment:
```suggestion
/**
* {{{.}}}
*/
```
How about we use actual java descriptions so ideas will pick up the description when devs hover over it?

Reply:
Could also add this atop the constructors/setters if we'd like to